spec.json

```diff
@@ -0,0 +1,74 @@
+{
+  "$ref": "#/definitions/event",
+  "definitions": {
+    "cloudEventsVersion": {
+      "type": "string"
+    },
+    "contentType": {
+      "type": "string"
+    },
+    "data": {
+      "type": [
+        "object",
+        "string"
+      ]
+    },
+    "event": {
+      "properties": {
+        "cloudEventsVersion": {
+          "$ref": "#/definitions/cloudEventsVersion"
+        },
+        "contentType": {
+          "$ref": "#/definitions/contentType"
+        },
+        "data": {
+          "$ref": "#/definitions/data"
+        },
+        "eventID": {
+          "$ref": "#/definitions/eventID"
+        },
+        "eventTime": {
+          "$ref": "#/definitions/eventTime"
+        },
+        "eventType": {
+          "$ref": "#/definitions/eventType"
+        },
+        "extensions": {
+          "$ref": "#/definitions/extensions"
+        },
+        "source": {
+          "$ref": "#/definitions/source"
+        }
+      },
+      "required": [
```

Comment:
Just thought I would feedback that we took the following JSON schema for our cloudevents API gateway endpoint and noticed these fields should be optional rather than required:

* eventTime
* contentType
* data
* extensions

Reply:
Excellent - thanks for checking this for us!!